src/main/java/com/example/project/repository/CartItemRepository.java

```diff
@@ -6,5 +6,5 @@
 import org.springframework.data.jpa.repository.JpaRepository;
 
 public interface CartItemRepository extends JpaRepository<CartItem, Long> {
-    List<CartItem> findAllByUsersId(Long userId, Pageable pageable);
+    List<CartItem> findAllByShoppingCart_User_Id(Long userId, Pageable pageable);
```

Comment:
we don't use _ in method names unless for tests, why did you change the name?

Reply:
we've discussed this issue. added changes in naming and methods using shoppingCartId approach